Model/ProductProtection.php

```diff
@@ -394,30 +430,88 @@ public function upsert(
     private function createOptions($product, $item, $updatedOptions): array
     {
         $options = [];
-        $optionIds = [];
-        foreach ($product->getOptions() as $o) {
-            $optionId = $o->getId();
-            $existingOption = $item->getOptionByCode("option_$optionId");
+        foreach (self::CUSTOM_OPTION_CODES as $optionCode) {
+            $existingOption = $item->getOptionByCode($optionCode);
             // if the option is in the updated options/values, create a new option
             // if there is no update to the option, use the existing option
-            if (isset($updatedOptions[$o->getTitle()])) {
+            if (isset($updatedOptions[$optionCode])) {
                 $option = $this->itemOptionFactory->create();
                 $option->setProduct($product);
-                $option->setCode("option_$optionId");
-                $option->setValue($updatedOptions[$o->getTitle()]);
+                $option->setCode($optionCode);
+                $option->setValue($updatedOptions[$optionCode]);
                 $options[] = $option;
-                $optionIds[] = $optionId;
             } elseif (isset($existingOption)) {
                 $options[] = $existingOption;
-                $optionIds[] = $optionId;
             }
         }
-        // build record of option ids for the product options that have values
-        $option = $this->itemOptionFactory->create();
-        $option->setProduct($product);
-        $option->setCode('option_ids');
-        $option->setValue(join(',', $optionIds));
-        $options[] = $option;
         return $options;
     }
+
+    /**
+     * Adds additional options to the product protection quote item
+     * These are used to display the product protection information on the storefront/admin
+     * They are also leveraged by the Magento SDK add-on (e.g. normalization)
+     *
+     * @param $item
+     * @param $productId
+     * @param $term
+     * @param $leadToken
+     * @param $leadQuantity
+     */
+    private function addAdditionalOptions($item, $productId, $term, $leadToken, $leadQuantity): void
+    {
+        $associatedProduct = $this->productRepository->get($productId);
+        if (!$associatedProduct) {
+            throw new LocalizedException(
+                new Phrase('Product with SKU %1 does not exist', $productId)
+            );
+        }
+
+        $numYears = $term / 12;
+        $termText = "{$numYears} years";
```

Comment:
Hardcoded language - something to just keep in mind when internationalization comes into play